docs/sources/clients/promtail/scraping.md

```diff
@@ -64,6 +64,29 @@ There are different types of labels present in Promtail:
   uniqueness of the streams. It is set to the absolute path of the file the line
   was read from.
 
+### Exemple of File Discovery
+```
```

Comment:
Could you add a sentence here that explains the example?  What are we doing here? How can the reader use this example?  Your PR description says this is for promtail configuration, but you don't actually say that in the documentation.